app/store/keys/keys.reducer.ts

```diff
@@ -0,0 +1,30 @@
+import { createReducer, createSelector } from '@reduxjs/toolkit';
+import log from 'electron-log';
+
+import { RootState } from '../reducers';
+import { persistMnemonicSafe, removeMnemonic, persistMnemonic } from './keys.actions';
+
+export interface KeysState {
+  mnemonic: string | null;
+}
```

Comment:
I would recommend only storing the STX private key. I suppose you need the mnemonic for onboarding, so the user can confirm, but after that you only need the STX private key. From there you can sign transactions, show address info, etc. Even once we figure out our story for persisting the encrypted private key, I'd recommend that we store the encrypted STX private key vs. the encrypted mnemonic. It's just simpler and more efficient.

You've probably already thought about this - just my two cents.

Reply:
Agreed, this only for onboarding. Figured we'll remove from the store once the PKs been generated. 

It would also be possible to not store the mnemonic at all and rederive the PK to compare the keys 🤔

Reply:
Oh true, hadn't even thought of that. Nice.